qiskit/providers/ibmq/job/queueinfo.py

```diff
@@ -56,5 +63,29 @@ def __init__(
         self.hub_priority = hub_priority
         self.group_priority = group_priority
         self.project_priority = project_priority
+        self.job_id = job_id
 
         super().__init__(**kwargs)
+
+    def format(self) -> str:
+        """Build an user-friendly report for the job queue information.
+
+        Returns:
+             The job queue information report.
+        """
+        job_status = api_status_to_job_status(ApiJobStatus(self._status))
+        estimated_start_time = arrow.get(self.estimated_start_time).humanize()
+        estimated_completion_time = arrow.get(self.estimated_complete_time).humanize()
+
+        queue_info = [
+            "Job {} queue information:".format(self.job_id),
+            "    queue position: {}".format(self.position),
+            "    status: {}".format(job_status.value),
+            "    estimated start time: {}".format(estimated_start_time),
+            "    estimated completion time: {}".format(estimated_completion_time),
+            "    hub priority: {}".format(self.hub_priority),
+            "    group priority: {}".format(self.group_priority),
+            "    project priority: {}".format(self.project_priority)
+        ]
```

Comment:
Technically any of these fields can be `None`. Can you change it to print `unknown` if the field is `None`?